awx/ui_next/src/screens/Inventory/InventorySources/InventorySourceSyncButton.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useCallback, useState, useEffect } from 'react';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Button, Tooltip } from '@patternfly/react-core';
+import { SyncIcon, MinusCircleIcon } from '@patternfly/react-icons';
+import useRequest, { useDismissableError } from '@util/useRequest';
+import AlertModal from '@components/AlertModal/AlertModal';
+import ErrorDetail from '@components/ErrorDetail/ErrorDetail';
+import { InventoryUpdatesAPI, InventorySourcesAPI } from '@api';
+
+function InventorySourceSyncButton({
+  onCancelSyncLoading,
+  onStartSyncLoading,
+  source,
+  i18n,
+}) {
+  const [updateStatus, setUpdateStatus] = useState(source.status);
+
+  const {
+    isLoading: startSyncLoading,
+    error: startSyncError,
+    request: startSyncProcess,
+  } = useRequest(
+    useCallback(async () => {
+      let syncStatus;
+
+      const {
+        data: { can_update },
+      } = await InventorySourcesAPI.allowSyncStart(source.id);
+      if (can_update) {
+        syncStatus = await InventorySourcesAPI.startSyncSource(source.id);
+      } else {
+        throw new Error(
```

Comment:
do we need this can_update/can_cancel conditionals.  It looks like we already hide the button based on user_capabilities start boolean.  I'm fine with simplifying this component with the assumption it is only rendered if they have that permission based on that, unless there's some reason why further checking is needed.

Reply:
spoke with @jlmitch5 offline.  It will be removed.